styles/stylesheet.css

```diff
@@ -0,0 +1,543 @@
+}
+
+@media screen and (max-width: 1279px) {
+    .hdr-right {
+        /* margin-top: 1rem; */
+        margin-right:2rem;
+    }
+}
+
+@media screen and (max-width: 575px) {
+    .hdr-right {
+        /* margin-top: .9rem; */
+        margin-right:1rem;
+    }
+}
+
+
+header p {
+    font-size: .9rem;
+    padding-top: 1rem;
+}
+
+header img,
+header p
+/* ,
+header button */ {
+    margin-top: .5%;
+}
+
+
+
+.cart {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-content: center;
+    font-size: .8rem;
+
+}
+
+.cart img {
+    width: 20%;
+    margin-bottom: 1em;
+}
+
+@media screen and (max-width: 575px) {
+    .cart img {
+        width: 25%;
+    }
+}
+
+@media screen and (max-width:1279px) {
+    .cart {
+        font-size: .5rem;
+    }
+}
+
+.cart p {
+    margin-left: 1rem;
+    padding-bottom: 1rem;
+}
+
+
+header button {
+    width: 100%;
+    display: block;
+    margin: 0 auto;
+    padding: .5rem 3rem .5rem 3rem;
+    background-color: #008A00;
+    border: 1px solid rgb(48, 44, 44);
+    color: white;
+    font-weight: bold;
+    font-size: .9rem;
+
+}
+
+@media screen and (max-width:1279px) {
+    header button {
+        /* width: 80%; */
+        padding: .5rem 2rem .5rem 2rem;
+    }
+}
+
+@media screen and (max-width:575px) {
+    header button {
+        /* width: 60%; */
+        font-size: .8rem;
+        padding: .5rem 2rem .5rem 2rem;
+    }
+}
+
+/* @media screen and (max-width: 991px) {
+    header button {
+        margin-left: 6em;
+    }
+} */
+
+nav {
+    background-color: lightgray;
+}
+
+nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-content: flex-start;
+    flex-wrap: wrap;
+    padding-left: 5%;
+}
+
+nav li,
+footer li,
+nav p {
+    list-style: none;
+    padding: 1rem 0rem 1rem 1rem;
+    font-size: .75rem;
+    font-weight: bold;
+}
+
+nav a,
+footer a {
+    text-decoration: none;
+}
+
+/* styling for home page */
+.wrapper-home,
+.wrapper {
+    margin-left: 10%;
+    margin-right: 10%;
+}
+
+.wrapper-home {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+/* styling for main content section on home page */
+.main-content {
+    width: 70%;
+}
+
+.intro {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-top: 3rem;
+    width: 100%;
+}
+
+.intro img {
+    max-width: 100%;
+    height: auto;
+}
+
+.dogs,
+.all-dogs {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(0, 32%));
+    grid-gap: 1% 1%;
+    margin-top: 1rem;
+    margin-bottom: 3rem;
+    overflow: auto;
+
+}
+
+.dogs article {
+    border: lightgray solid 1px;
+}
+
+@media screen and (max-width: 575px) {
+    .dogs,
+    .all-dogs {
+        grid-template-columns: repeat(auto-fit, minmax(0rem, 100%));
+    }
+}
+
+@media screen and (min-width: 576px) and (max-width: 1279px) {
+    .dogs,
+    .all-dogs {
+        grid-template-columns: repeat(auto-fit, minmax(0rem, 45%));
```

Comment:
Great work with the media queries. Keep in mind you can have multiple css queries within a media query.